<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraceWalkerDialogBase</class>
 <widget class="QDialog" name="TraceWalkerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TraceWalker</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>歩行スピード（km/h)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="value">
    <double>4.800000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>体重（kg)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="excludedProviders">
    <stringlist notr="true"/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>設定</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>button_box</zorder>
  <zorder>label</zorder>
  <zorder>doubleSpinBox_1</zorder>
  <zorder>label_2</zorder>
  <zorder>doubleSpinBox_2</zorder>
  <zorder>label_4</zorder>
  <zorder>mMapLayerComboBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
